flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -873,8 +873,7 @@ class JobManager(
       }
 
       sender ! decorateMessage(
-        PartitionState(
-          taskExecutionId,
+        new org.apache.flink.runtime.io.network.PartitionState(
```

Comment:
I think it's not ambiguous anymore after removal of the TaskControlMessage and we can directly use `PartitionState`?